create_derivatives/tests.py

```diff
@@ -118,3 +142,42 @@ def test_run(self, mock_get_object, mock_init):
     def tearDown(self):
         for f in Path(settings.SRC_DIR).iterdir():
             f.unlink()
+
+
+class AWSUploadTestCase(TestCase):
+
+    def setUp(self):
+        """Creates a temp directory and adds an uncompressed bag fixture to the temp directory and database"""
+        path = Path(settings.TMP_DIR)
+        if not path.exists():
+            path.mkdir(parents=True)
+        bag = "3aai9usY3AZzCSFkB3RSQ9"
+        fixture_directory = "aws_upload_bag"
+        bag_path = Path(settings.TMP_DIR, bag)
+        if not Path(bag_path).exists():
+            shutil.copytree(Path("create_derivatives", "fixtures", fixture_directory, bag), bag_path)
+            Bag.objects.create(
+                bag_identifier="sdfjldskj",
+                bag_path=bag_path,
+                origin="digitization",
+                as_data="sdjfkldsjf",
+                dimes_identifier=bag,
+                process_status=Bag.MANIFESTS_CREATED)
+
+    @patch("create_derivatives.clients.AWSClient.__init__")
+    @patch("create_derivatives.clients.AWSClient.upload_files")
+    def test_run(self, mock_upload_files, mock_init):
+        """Asserts that the run method produces the desired results message.
+
+        Tests that the method updates the bag's process_status at the end.
+        """
+        mock_init.return_value = None
+        routine = AWSUpload()
+        file_upload = routine.run(True)
+        self.assertEqual(file_upload[0], "Bags successfully uploaded")
+        self.assertTrue(file_upload)
```

Comment:
```suggestion
        msg, object_list = routine.run(True)
        self.assertEqual(msg, "Bags successfully uploaded")
        self.assertTrue(isinstance(object_list, list))
        self.assertEqual(len(object_list), 1)
```
that `assertTrue` on the return isn't doing much.